.github/workflows/build.yml

```diff
@@ -35,6 +35,13 @@ jobs:
         with:
           path: ./source
           submodules: true
+      - name: Set up GCS Key
+        run: echo $GCS_KEY > /tmp/gcs_key.json
+        shell: bash
+        env:
+          GCS_KEY: ${{secrets.SCCACHE_GCS_KEY}}
+      - name: Set up sccahe
+        run: echo "/root/.cargo/bin" >> $GITHUB_PATH
```

Comment:
Can this be done as part of the docker container build? It's just adding to the path right?